lab_4_summarization_textrank/main.py

```diff
@@ -160,22 +231,27 @@ def __init__(self) -> None:
         """
         Constructs necessary attributes
         """
-        pass
+        self._matrix = []
+        self._vertices = []
 
     def get_vertices(self) -> tuple[Sentence, ...]:
         """
         Returns a sequence of all vertices present in the graph
         :return: a sequence of vertices
         """
-        pass
+        return tuple(self._vertices)
 
     def calculate_inout_score(self, vertex: Sentence) -> int:
         """
         Retrieves a number of vertices that are similar (i.e. have similarity score > 0) to the input one
         :param vertex
         :return:
         """
-        pass
+        check_type(vertex, Sentence, False)
+        if vertex not in self._vertices:
+            raise ValueError
+        idx = self._vertices.index(vertex)
+        return len([element for element in self._matrix[idx] if element > 0])
```

Comment:
[] is redundant

Reply:
TypeError raises without [] 

File "D:\2022-2-level-labs\lab_4_summarization_textrank\main.py", line 263, in calculate_inout_score
    return len(element for element in self._matrix[idx] if element > 0)
TypeError: object of type 'generator' has no len()